### DISCUSSION TITLE
研究生盲审模式修改

### DISCUSSION THREAD

**Discussion by dreistein543** [nju-lug/NJUThesis#180] [Q&A]
通过最新版(v1.1.2)模板打开盲审模式时，在摘要页依然保留了一些信息如下图所示
<img width="535" alt="image" src="https://user-images.githubusercontent.com/30745407/222181715-a223aca7-13bb-4c55-af49-7e470db5ae95.png">

<img width="538" alt="image" src="https://user-images.githubusercontent.com/30745407/222181855-aa9790c6-4e72-429f-9335-6808646bf15f.png">

先提一下，这两天问一下教务员再来确认是否确实需要剔除

**Comment by atxy-blip** [comment]
<https://github.com/nju-lug/NJUThesis/commit/41a8982397cb28f3741d2d8f73ca1ddb9f0bf762> 修了。

**Comment by dreistein543** [comment]
最近教务系统开放了盲审的格式，可能需要对盲审模式进行一定的更新：
1. 不需要隐去“南京大学”和“校徽”
2. 将作者和导师替换为“XXX”
3. 成果仅展示年份和期刊
4. 致谢页显示“致谢 (盲审阶段，暂时隐去)”
具体样式请参考[匿名论文样本.pdf](https://github.com/nju-lug/NJUThesis/files/10967296/default.pdf)

**Reply by atxy-blip** [reply]
收到，最近有空时会去改改。

**Reply by dreistein543** [reply]
辛苦，由于3月25号就要提交，我只能自己在网页上新建一个盲审版本了

**Reply by AllenDon017** [reply]
麻烦问一下，盲审模式改了吗？24届也是按照这个匿名论文样本.pdf来要求的，我尝试在cls文件里注释掉了下面两段：
\bool_if:NT \g__nju_opt_anon_bool
  {
    \tl_clear:N \l__nju_name_nju_tl
    \tl_clear:N \l__nju_name_nju_en_tl
  }
  {
        \RenewDocumentCommand \njuemblem { o m m } { }
        \RenewDocumentCommand \njuname   { o m m } { }
      }
可以显示校徽校名了，但是不知道怎么讲人名替换成”XXX“，以及可能还有其他的一些具体样式问题。

**Reply by atxy-blip** [reply]
@AllenDon017 一直拖着没改，明天我处理一下吧。

**Reply by atxy-blip** [reply]
改了，见 [98aebbd](https://github.com/nju-lug/NJUThesis/commit/98aebbd10392f1773f43fb6dcc7a082af144ca3a)。
